src/dllmain.cpp

```diff
@@ -1134,6 +1134,28 @@ void ViewportFix()
     }
 }
 
+// cipherxof's Skybox Rendering Fix
+void SkyboxFix()
+{
+    if (eGameType != MgsGame::MGS2)
+    {
+        return;
+    }
+
+    uintptr_t MGS2_CreateSkyUtilScanResult = (uintptr_t)Memory::PatternScan(baseModule, "81 4F 58 00 30 00 00");
```

Comment:
```suggestion
    uintptr_t MGS2_CreateSkyUtilScanResult = (uintptr_t)Memory::PatternScan(baseModule, "81 4F ?? ?? ?? ?? ?? 4D 85 FF");
```
my sig maker is saying this is the shortest unique. (the one you have overlaps with 3 other functions)

Reply:
we know the value is 0x3000 and the offset is 0x58. These are very unlikely to change (especially the value) so I suppose at best it would be `81 4F ?? ?? 30 00 00 4D 85 FF`